plugins/micromega/micromega.ml

```diff
@@ -1,3 +1,4 @@
+module OCaml = struct type ocaml_int = int end
```

Comment:
This triggers fatal warnings because it's unused.
If it's easy to detect the unused module we could avoid outputting it, otherwise we can output `[@@@warnings "-60-34"]` before it.

Reply:
To detect if used, it would almost be the proposal I made [above], which I believe is more to be considered in the hands of @coq/extraction-maintainers. 

To avoid the warning, I guess it would be enough to add a declaration of the new module also in the non-generated file `micromega.mli`??

Reply:
I don't think we want it in the mli though

Reply:
You mean "we want it in micromega.mli" or "we want it in the mli files in general".

If the second, this does not correspond to the needs. We have `mli`s that need to refer to OCaml `int` while having Coq `int` in their scope.

Reply:
That's true